turbolinks/build.gradle

```diff
@@ -1,46 +1,47 @@
 plugins {
-    id 'com.android.library'
-    id 'kotlin-android'
+  id 'com.android.library'
+  id 'kotlin-android'
 }
 
 android {
-    compileSdkVersion rootProject.ext.compileSdkVersion
-    defaultConfig {
-        minSdkVersion rootProject.ext.minSdkVersion
-        targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 11
-        versionName "1.0.9"
+  compileSdkVersion rootProject.ext.compileSdkVersion
+  defaultConfig {
+    minSdkVersion rootProject.ext.minSdkVersion
+    targetSdkVersion rootProject.ext.targetSdkVersion
+    versionCode 11
+    versionName "1.0.9"
 
-        // Define ProGuard rules for this android library project. These rules will be applied when
-        // a consumer of this library sets 'minifyEnabled true'.
-        consumerProguardFiles 'proguard-consumer-rules.pro'
-    }
-    buildFeatures {
-        viewBinding true
-    }
-    buildTypes {
-        release {
-            minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
-        }
-    }
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_11
-        targetCompatibility JavaVersion.VERSION_11
+    // Define ProGuard rules for this android library project. These rules will be applied when
+    // a consumer of this library sets 'minifyEnabled true'.
+    consumerProguardFiles 'proguard-consumer-rules.pro'
+  }
+  buildFeatures {
+    viewBinding true
+  }
+  buildTypes {
+    release {
+      minifyEnabled false
+      proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
     }
+  }
+  compileOptions {
+    targetCompatibility rootProject.ext.targetJvmVersion
+    sourceCompatibility rootProject.ext.sourceJvmVersion
+  }
+  namespace "com.basecamp.turbolinks"
```

Comment:
The only actual change introduced in this PR is this chunk here. The rest is an indentation change.